src/components/quickstarts/QuickstartOverview.js

```diff
@@ -38,24 +38,24 @@ const QuickstartOverview = ({ quickstart }) => {
         {quickstart.dashboards.map((dashboard, index) => (
           <OverviewTile
             key={index}
-            title={dashboard.name}
-            image={dashboard.screenshots[0]}
+            title={dashboard.displayName}
+            image={dashboard.previews[0].url}
```

Comment:
It's possible for a quickstart dashboard to not have a preview image and in that case this will break. Example: [data-ingest-breakdown](https://github.com/newrelic/newrelic-quickstarts/tree/main/quickstarts/audit/account-data-ingestion-analysis)